pylintrc_backup

```diff
@@ -0,0 +1,387 @@
+# ***************************
+# ** DO NOT EDIT THIS FILE **
```

Comment:
Also, not sure if you can do this for all your PRs that have this problem.
FYI: @nedbat

Reply:
Ned brought up that this is a temporary file that isn't meant to be committed, and we discussed possible solutions around this using `.gitignore`.

Reply:
Definitely don't commit this file.  Ideally use it to figure out what previous hand-edited change needs to be moved to a pylintrc_tweaks file.

I'm conflicted about whether to add it to .gitignore

Reply:
@nedbat: If we definitely don't want to commit this file, the reason to add it to .gitignore is clear. Can you provide more details on the downside of adding it to .gitignore that has you conflicted?

Reply:
@sarina: We could always loop back and add to .gitignore later, so maybe you can just delete this file for now so we can get this PR through. Thanks!

Reply:
The reason I might leave it out of .gitignore is that it is not an expected artifact of normal operations.  As Sarina says, it's a bit of a warning. So I'd leave it more visible.

Reply:
As-is, I think people will not understand what it is for and it will get committed and possibly merged. If you are thinking about additional checks around all this, something that failed the build if this file is found, with a clear message about why and how to fix would be useful. Again, as-is, I don't think you'll get what you want.

Reply:
I agree that simply letting the file get committed will not get what I want.  I'm looking over possibilities for checking for the file automatically as a linting operation.